api_docs/unmerged.d/ZF-01a225

```diff
@@ -0,0 +1,3 @@
+* [`DELETE /users/{user_id}`](/api/deactivate-user): Added new
+  parameters `is_spammer` and `message_delete_action` for marking 
```

Comment:
`message_delete_action` parameter is used to delete messages irrespective of whether user is being marked as spammer or not.